concordia/templates/transcriptions/campaign_list.html

```diff
@@ -13,28 +13,24 @@
 {% endblock breadcrumbs %}
 
 {% block main_content %}
-<div class="container-fluid py-3">
-    <div class="row">
-        <div class="d-flex flex-wrap col-12 mb-3 justify-content-around">
-            {% for campaign in campaigns %}
-                <div class="bg-lightest-gray w-100 mb-3">
-                    <div class="row">
-                        <div class="col-12 col-md-8 px-3">
-                            <a href="{% url 'transcriptions:campaign-detail' campaign.slug %}">
-                                <h1 class="campaign-title">{{ campaign.title }}</h1>
-                            </a>
-                            {{ campaign.short_description|linebreaksbr }}
-                        </div>
-                        <div class="col-12 col-md-4">
-                            <a href="{% url 'transcriptions:campaign-detail' campaign.slug %}">
-                            <img src="{{ MEDIA_URL }}{{ campaign.thumbnail_image }}" class="img-fluid" alt="{{ campaign.title }} image">
-                            </a>
-                        </div>
-                    </div>
+<div class="container py-3">
```

Comment:
Is intentional not to follow the usual `container` / `row` hierarchy? It seems like we should wrap the list in a row so it would be a peer of the pagination block below.

Reply:
It is intentional, as `row` is only needed when you're creating columns. However `row`s negative left/right margin is useful for a container inside `container` to expand to the edge. And since bootstrap grid gutter is based in px (30px) and their spacing is base in rem, lining things up is not easy. I'd like to try to fix this once we implement bootstrap scss.

Reply:
You're referring to the spacing using the padding class? My usual approach for this is to use the container/row with no margin/padding on those elements so the margin values will match (both use `px`) and then set a margin or padding on the the contents so they'll align consistently.

Either way, this seems like a great thing to defer until we reconsider our front-end toolchain.